aisdc/attacks/worst_case_attack.py

```diff
@@ -218,7 +220,7 @@ def run_attack_reps(  # pylint: disable = too-many-locals
         )
 
         mia_metrics = []
-        for rep in range(self.args.n_reps):
+        for rep in range(1, self.args.n_reps + 1, 1):
```

Comment:
I think we should change this -- if the issue is just in the output, just add 1 to the output.

Reply:
Agreed - will update this.